react/game/src/app/screens/Game/layout.js

```diff
@@ -26,7 +26,7 @@ class GameLayout extends Component {
           winner={this.props.winner}
         />
         <div className={styles.gameInfo}>
-          <div className={`${styles.status} ${this.getClassWinner}`}>{status}</div>
+          <div className={`${styles.status} ${this.getClassWinner()}`}>{status}</div>
```

Comment:
Why this `()`?

Reply:
Because of this:
![image](https://user-images.githubusercontent.com/15016271/46759720-b265a780-cc95-11e8-83a0-ec22832ab44f.png)

We are just assigning a function, not calling it!